From 09da66c99a6be81c5701faae03324f7f9b335d3c Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 17 Jan 2015 17:45:27 -0800
Subject: [PATCH 2003/2005] Fall back on xcodebuild -sdk when xcrun --sdk is
 not supported

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 4dc615877d820549a0df45b1ba8bd5067fac7a85)
---
 make/platform/clang_darwin.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git llvm_248522-branch/projects/compiler-rt/make/platform/clang_darwin.mk macports_248522-branch/projects/compiler-rt/make/platform/clang_darwin.mk
index 2bab6c8..e5a81b0 100644
--- llvm_248522-branch/projects/compiler-rt/make/platform/clang_darwin.mk
+++ macports_248522-branch/projects/compiler-rt/make/platform/clang_darwin.mk
@@ -52,6 +52,10 @@ XCRunSdkPath = \
     result=`xcrun --sdk $(1).internal --show-sdk-path 2> /dev/null`; \
     if [ "$$?" != "0" ]; then \
       result=`xcrun --sdk $(1) --show-sdk-path 2> /dev/null`; \
+      if [ "$$?" != "0" ]; then \
+        result=`xcodebuild -sdk $(1) -version 2> /dev/null \
+                | sed -n 's/^Path: \(..*\)$$/\1/p'`; \
+      fi; \
       if [ "$$?" != "0" ]; then result=""; fi; \
     fi; \
     echo $$result)
-- 
2.6.1

